code/modules/mob/living/simple_animal/bot/medbot.dm

```diff
@@ -182,8 +189,22 @@ GLOBAL_VAR(medibot_unique_id_gen)
 		heal_threshold += adjust_num
 		if(heal_threshold < 5)
 			heal_threshold = 5
-		if(heal_threshold > 75)
-			heal_threshold = 75
+		if(heal_threshold > 120)
+			heal_threshold = 120
+
+	else if(href_list["adj_inject"])
+		var/adjust_num = text2num(href_list["adj_inject"])
+		injection_amount += adjust_num
+		if(injection_amount < 5)
+			injection_amount = 5
+		if(injection_amount > 15)
+			injection_amount = 15
```

Comment:
I and Ruko agreed with increasing the max from 15u to more. 30u will be enough.
We want to see a conflict from a case that a dumb person didn't think about OD